index.js

```diff
@@ -188,7 +164,7 @@ function sendNotification(endpoint, params) {
         options.headers['Authorization'] = 'Bearer ' + jwt;
         var key = 'p256ecdsa=' + urlBase64.encode(vapid.publicKey);
         if (options.headers['Crypto-Key']) {
-          options.headers['Crypto-Key'] += ',' + key;
+          options.headers['Crypto-Key'] += ';' + key;
```

Comment:
Yes, semicolons separate parameters, commas separate values.

From http://httpwg.org/http-extensions/draft-ietf-httpbis-encryption-encoding.html#crypto-key:
> The Crypto-Key header field uses the extended ABNF syntax defined in Section 1.2 of [RFC7230]

From https://tools.ietf.org/html/rfc7230#section-1.2:
> This specification uses the Augmented Backus-Naur Form (ABNF) notation of [RFC5234] with a list extension, defined in Section 7, that allows for compact definition of comma-separated lists [...]

Then, in https://tools.ietf.org/html/draft-ietf-webpush-vapid-00#section-4:
> Note that with push message encryption [I-D.ietf-webpush-encryption], this results in two values in the Crypto-Key header field, one with the a "p256dh" key and another with a "p256ecdsa" key.

So, in this case, I think it should be a comma.